depends/packages/zeromq.mk

```diff
@@ -6,7 +6,8 @@ $(package)_sha256_hash=bcbabe1e2c7d0eec4ed612e10b94b112dd5f06fcefa994a0c79a45d83
 $(package)_patches=0001-fix-build-with-older-mingw64.patch 0002-disable-pthread_set_name_np.patch
 
 define $(package)_set_vars
-  $(package)_config_opts=--without-docs --disable-shared --without-libsodium --disable-curve --disable-curve-keygen --disable-perf --disable-Werror
+  $(package)_config_opts=--without-docs --disable-shared --disable-curve --disable-curve-keygen --disable-perf --disable-Werror --disable-drafts
+  $(package)_config_opts += --without-libsodium --without-libgssapi_krb5 --without-pgm --without-norm --without-vmci
```

Comment:
Looking at https://gist.github.com/dongcarl/892d7deda8da3ff72317009ecbb07ffb#file-zeromq-configure-help-L114-L120, I'm reminded that after we're able to use Guix for release builds, we will finally be able to, and definitely should specify what the minimal kernel headers are that we target for each architecture. We don't want to target kernel headers that are too new, have the zeromq build auto-assume we have the fancy new `ekquelect` (not an actual thing) available, and then not be able to run on older kernels.

Reply:
Is this something you'd like to document here, or should we start tracking future Guix related improvements somewhere else? Depends how much there is to keep track of.

Reply:
Tracked https://github.com/bitcoin/bitcoin/issues/16496